docs/changelog.rst

```diff
@@ -17,6 +17,7 @@ New
 
 Changed
 -------
+Allow masking in `GridComponent.set`. (#1229)
```

Comment:
Nitpicky but could count as Added as this is new functionnality from that function.